### PR TITLE
add grafana panel for tiproxy

### PR DESCRIPTION
**Pull request by djshow832**
Add the grafana panel for tiproxy.

The tag for TiProxy is different with that of TiDB, so always fetch the latest one.

### CODE REVIEW

File: jenkins/pipelines/cd/tiup/prometheus-tiup-mirror-update.groovy [hunk 1]
```diff
@@ -96,7 +96,8 @@ def pack = { version, os, arch ->
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/blacker.rules.yml || true; \
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/bypass.rules.yml || true; \
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/kafka.rules.yml || true; \
-    wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/node.rules.yml || true;
+    wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/rule/node.rules.yml || true; \
+    wget -qnc https://raw.githubusercontent.com/pingcap/tiproxy/main/pkg/metrics/alertmanager/tiproxy.rules.yml || true;
```
Comment (lijie0123) [inline jenkins/pipelines/cd/tiup/prometheus-tiup-mirror-update.groovy:101]: it's better to use https://raw.githubusercontent.com/pingcap/tiproxy/${tag}/pkg/metrics/alertmanager/tiproxy.rules.yml

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy [hunk 2]
```diff
@@ -53,6 +53,7 @@ def pack = { version, os, arch ->
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/grafana/blackbox_exporter.json || true; \
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/grafana/node.json || true; \
     wget -qnc https://raw.githubusercontent.com/pingcap/monitoring/master/platform-monitoring/ansible/grafana/kafka.json || true; \
+    wget -qnc https://raw.githubusercontent.com/pingcap/tiproxy/main/pkg/metrics/grafana/tiproxy_summary.json || true; \
```
Comment (lijie0123) [inline jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy:56]: we are now not using this script, edit https://github.com/pingcap/monitoring/blob/master/monitoring.yaml please

**Review by lijie0123 — CHANGES_REQUESTED** [review]

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy (hunk 2 above)
Comment (wuhuizuo) [reply jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy:56]: No need to make change to this file.

File: jenkins/pipelines/cd/tiup/prometheus-tiup-mirror-update.groovy (hunk 1 above)
Comment (wuhuizuo) [reply jenkins/pipelines/cd/tiup/prometheus-tiup-mirror-update.groovy:101]: Just main branch in `tiproxy`, no release branch exist.

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy (hunk 2 above)
Comment (wuhuizuo) [inline jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test.groovy:56]:
```suggestion
```

**Review by wuhuizuo — APPROVED** [review]
/lgtm
/approve

**Pull request merged**